app/globals.css

```
@@ -84,12 +294,80 @@
+    page-break-inside: avoid;
+  }
+
+  .cv-skill-category h3 {
+    font-size: 12pt;
+    font-weight: bold;
+    margin-bottom: 6pt;
+    color: #333 !important;
+  }
+
+  /* Ensure all content is visible and structured for ATS */
+  .print\\:block {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-class block 

See MDN web docs for more details.

(lint/correctness/noUnknownPseudoClass)